jinahub/segmenters/VADSpeechSegmenter/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM jinaai/jina:latest-perf
+
+# install git
+RUN apt-get -y update && apt-get install -y git
```

Comment:
Don't need anymore, am i right?
```suggestion
```

Reply:
Yes you are right, `apt-get install git` is not needed anymore, thank you.